content/en/guide/v10/typescript.md

```diff
@@ -81,7 +81,7 @@ type Props = {
   age: number;
 };
 
-function MyComponent({ name, age }: Props) {
```

Comment:
Most React library use  `React.FC`.
In Preact, most library use `FunctionComponent`.
Using `FunctionComponent` method is general.
Therefore, This example is unnecessary.